lib/mix/support/system_command_helper.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Mix.Support.SystemCommandHelper do
+  def run_system_command(command) when is_binary(command) do
+    command
+    |> String.to_charlist()
+    |> run_system_command()
+  end
+
+  def run_system_command(command) do
+    response =
+      command
+      |> :os.cmd()
+      |> to_string()
+
+    if response != "" do
```

Comment:
This really only works for system commands where not output is a success (like `cp`). I'm sure there's a better way to do this, but I figure it's good enough for now.